awx/main/notifications/webhook_backend.py

```diff
@@ -75,7 +72,7 @@ def send_messages(self, messages):
                 "{}".format(m.recipients()[0]),
                 auth=auth,
                 data=json.dumps(m.body, ensure_ascii=False).encode('utf-8'),
-                headers=self.headers,
+                headers=get_awx_http_client_headers(),
```

Comment:
Actual code change, but should do the right thing here (didn't want to cut & paste the user agent templating)